### PR TITLE
fix: synpress

### PR DESCRIPTION
**Pull request by simboonlong**
- consistent failing since `^3.7.1` update.
- recent fresh wave of [issues](https://github.com/Synthetixio/synpress/issues?q=is%3Aissue+created%3A2023-06-13..2023-07-13+).
- state of [confirmTransaction](https://github.com/Synthetixio/synpress/blob/a125d96d7f3cca6bfbcc71f660775b2fd7b2a8e2/commands/metamask.js#L886-L1101).
- reverting to last known working version `^3.5.1` in this repo.
- refactored:
  - explicit ordering by renaming filename of sequential tests.
  - more generous wait duration.
  - shift all metamask account switching to `before`.

### CODE REVIEW

File: tests/e2e/support/commands.js [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-    if ($transferPanel.find("[data-testid='connectToWallet']").length) {
-      cy.get("[data-testid='connectToWallet']").click();
-      if (acceptMetamask) {
-        cy.acceptMetamaskAccess({ allAccounts: true }).then((connected) => {
-          expect(connected).to.be.true;
-        });
-        cy.wait(2000);
-      }
-    }
+const METAMASK_WAIT = process.env.METAMASK_WAIT || 3000;
```
Comment (osslgtm) [inline tests/e2e/support/commands.js]: can `cy.log(METAMASK_WAIT)` ?
had to use `Cypress.env("ACCOUNT_2_PK")` to access env var of CYPRESS_ACCOUNT_2_PK

Comment (simboonlong) [reply tests/e2e/support/commands.js]: sure let me check

Comment (simboonlong) [reply tests/e2e/support/commands.js]: direct 5s instead

**Review by osslgtm — APPROVED** [review]
LGTM

**Pull request merged**